.babelrc

```diff
@@ -0,0 +1,3 @@
+{
+  "plugins": ["@babel/plugin-syntax-bigint"]
```

Comment:
Can you explain to dumb old me why this is needed? If the tests are running in node >=10.13.0 I would expect them to work just fine with BigInt, but maybe Jest is doing something magical behind the scenes.

Reply:
Yup, you guessed it right! Jest needs it so then it can read what `12n` means, for example.